examples/with-redux-toolkit/store.js

```diff
@@ -1,26 +1,12 @@
-import { configureStore, createReducer } from '@reduxjs/toolkit'
+import { configureStore } from '@reduxjs/toolkit';
 
-const initialState = {
-  light: false,
-  lastUpdate: 0,
-  count: 0,
-}
+import clockReducer from './lib/slices/clockSlice';
+import countReducer from './lib/slices/counterSlice';
 
-const reducer = createReducer(initialState, {
-  TICK: (state, action) => ({
-    ...state,
-    lastUpdate: action.payload.lastUpdate,
-    light: !!action.light,
-  }),
-  INCREMENT: (state, action) => ({ ...state, count: state.count + 1 }),
-  DECREMENT: (state, action) => ({ ...state, count: state.count - 1 }),
-  RESET: (state, action) => ({ ...state, count: initialState.count }),
-})
-
-const initializeStore = (preloadedState = initialState) => {
-  return configureStore({
-    reducer,
-    preloadedState,
-  })
-}
-export const store = initializeStore()
+export default configureStore({
+  reducer: {
+    count: countReducer,
```

Comment:
I would prefer `counter` & `counterReducer`

Reply:
Do it make any difference?

Reply:
Nope, just clearer IMHO :)

Reply:
The slice is named `counter`, I would expect their names are matched.